examples/practice-typescript/src/components/TableHeader /index.tsx

```diff
@@ -0,0 +1,40 @@
+import User from 'scripts/types/user';
+import './index.scss';
+import Product from 'scripts/types/product';
+
+interface HeaderItemProps {
+  label: string;
+}
+
+const HeaderItem = ({ label }: HeaderItemProps) => {
+  const formatLabel = (str: string) => {
+    return str
+      .replace(/\b\w/g, (char) => char.toUpperCase())
+      .replace(/([A-Z])/g, ' $1')
+      .trim();
+  };
+
+  return <th className="table-head">{formatLabel(label)}</th>;
+};
+
+interface TableHeaderProps {
+  headers: User[] | Product[];
+}
+
+const TableHeader = ({ headers = [] }: TableHeaderProps) => {
+  const headerKeys = Object.keys(headers[0]).filter(
+    (header) => header !== 'id'
+  );
```

Comment:
I don't get this

Reply:
Fixed and I want to get the keys in the array in the first position such as email, first name, last name without taking the id and save it in an array.

Reply:
Based on your explanation, this one was right but need to rename the callback should be `(key) => key !== 'id'`. The filter should belong here and the HeaderItem should only render

Reply:
Fixed